src/client/pages/OfferNew/Checkout/CreditCheckInfo.tsx

```diff
@@ -0,0 +1,50 @@
+import styled from '@emotion/styled'
+import { colorsV2, colorsV3 } from '@hedviginsurance/brand'
+import React from 'react'
+import { FadeIn } from 'components/animations/appearings'
+import { InfoIcon } from 'components/icons/Info'
+import { useTextKeys } from 'utils/textKeys'
+
+const InfoBox = styled.div`
+  padding: 1rem;
+  padding-left: 3rem;
+  margin-bottom: 1.5rem;
+  background-color: ${colorsV2.violet100};
+  position: relative;
+  border-radius: 8px;
+  svg {
+    position: absolute;
+    left: 1.125rem;
+    top: 1.125rem;
+  }
+`
+
+const InfoHeader = styled.p`
+  color: ${colorsV3.gray900};
+  margin: 0;
+  line-height: 1.5rem;
+`
+const InfoContent = styled.p`
+  color: ${colorsV3.gray700};
+  margin: 0;
+  margin-top: 0.25rem;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+`
```

Comment:
We try to follow the pattern of content being "pushed down" and using `margin-bottom`  :) 
```suggestion
const InfoHeader = styled.p`
  color: ${colorsV3.gray900};
  margin-top: 0;
  margin-bottom: 0.25rem;
  line-height: 1.5rem;
`
const InfoContent = styled.p`
  color: ${colorsV3.gray700};
  margin: 0;
  font-size: 0.875rem;
  line-height: 1.25rem;
`
```

Reply:
Okay, I'm used to the other way around. What's nice when the following component owns the spacing is that you don't have to worry about a leftover spacing at the end

Reply:
Haha, do we? I'd say we shouldn't use `margin` at all but go for padding and spacer components but this is a way bigger discussion 🙈